### DISCUSSION TITLE
Language translation

### DISCUSSION THREAD

**Discussion by Nicolo-R** [apache/superset#17031] [Q&A / Help]
Community hello!

Please tell me. Is there a translation file for Superset in Russian?

If not, which file is responsible for the information for the visual panel?

Thanks.

**Comment by Yellow-BringtEast** [comment]
in superset-v1.3.0, there is a config file called config.py in superset package in which you can change the language you want.
![image](https://user-images.githubusercontent.com/66516223/136521241-6e593c04-8812-4980-ab8e-67b5eb982fd2.png)
